src/lib/Transport/index.ts

```diff
@@ -116,11 +116,14 @@ export default class Transport {
           : (`${req.headers.referer}?userId=27&customer=dev&locale=en-US&platform=desktop&context=${context.id}`);
 
         const headers = {
+          ...req.headers,
           ...this.client.getAuthHeader(),
-          'Content-Type': req.headers['content-type'] || req.headers['Content-Type'] || 'application/json',
-          Referer: referer,
+          referer,
         };
 
+        // delete original headers that don't mess with service
```

Comment:
should this say `that DO mess with the service`? Just trying to make sure I understand here